src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-            "extensions": {
-                "VK_EXT_descriptor_indexing": 2,
-                "VK_EXT_external_memory_host": 1,
-                "VK_EXT_host_query_reset": 1,
-                "VK_EXT_image_robustness": 1,
-                "VK_EXT_inline_uniform_block": 1,
-                "VK_EXT_memory_budget": 1,
-                "VK_EXT_pipeline_creation_cache_control": 3,
-                "VK_EXT_private_data": 1,
-                "VK_EXT_robustness2": 1,
-                "VK_EXT_scalar_block_layout": 1,
-                "VK_EXT_separate_stencil_usage": 1,
-                "VK_EXT_shader_atomic_float": 1,
-                "VK_EXT_subgroup_size_control": 2,
-                "VK_EXT_texel_buffer_alignment": 1,
-                "VK_EXT_vertex_attribute_divisor": 3,
-                "VK_KHR_16bit_storage": 1,
-                "VK_KHR_8bit_storage": 1,
-                "VK_KHR_bind_memory2": 1,
-                "VK_KHR_buffer_device_address": 1,
-                "VK_KHR_copy_commands2": 1,
-                "VK_KHR_create_renderpass2": 1,
-                "VK_KHR_dedicated_allocation": 3,
-                "VK_KHR_depth_stencil_resolve": 1,
-                "VK_KHR_descriptor_update_template": 1,
-                "VK_KHR_device_group": 4,
-                "VK_KHR_driver_properties": 1,
-                "VK_KHR_dynamic_rendering": 1,
-                "VK_KHR_external_fence": 1,
-                "VK_KHR_external_memory": 1,
-                "VK_KHR_external_semaphore": 1,
-                "VK_KHR_get_memory_requirements2": 1,
-                "VK_KHR_image_format_list": 1,
-                "VK_KHR_imageless_framebuffer": 1,
-                "VK_KHR_maintenance1": 2,
-                "VK_KHR_maintenance2": 1,
-                "VK_KHR_maintenance3": 1,
-                "VK_KHR_multiview": 1,
-                "VK_KHR_push_descriptor": 2,
-                "VK_KHR_relaxed_block_layout": 1,
-                "VK_KHR_sampler_mirror_clamp_to_edge": 3,
-                "VK_KHR_sampler_ycbcr_conversion": 14,
-                "VK_KHR_separate_depth_stencil_layouts": 1,
-                "VK_KHR_shader_draw_parameters": 1,
-                "VK_KHR_shader_float16_int8": 1,
-                "VK_KHR_shader_float_controls": 4,
-                "VK_KHR_shader_subgroup_extended_types": 1,
-                "VK_KHR_spirv_1_4": 1,
-                "VK_KHR_storage_buffer_storage_class": 1,
-                "VK_KHR_swapchain": 70,
-                "VK_KHR_swapchain_mutable_format": 1,
-                "VK_KHR_timeline_semaphore": 2,
-                "VK_KHR_uniform_buffer_standard_layout": 1,
-                "VK_KHR_variable_pointers": 1
-            },
-            "features": {
-                "VkPhysicalDeviceFeatures": {
-                    "depthBiasClamp": true,
-                    "depthClamp": true,
-                    "drawIndirectFirstInstance": true,
-                    "dualSrcBlend": true,
-                    "fillModeNonSolid": true,
```

Comment:
did we use to require this

Reply:
yes we do require `fillModeNonSolid` and it's available on all profiles